src/renderer/containers/map/mapClass.jsx

```diff
@@ -105,7 +112,10 @@ const MapComponent = compose(
     </div>
     {props.isMarkerShown && (
       <Marker
-        position={{ lat: 39.9611755, lng: -82.99879420000002 }}
+        position={{
```

Comment:
we probably do not want the users current location to be represented by a marker that looks like the other markers.